lib/releaselib.sh

```diff
@@ -450,6 +450,15 @@ release::gcs::copy_release_artifacts() {
   local gce_path=$release_stage/full/kubernetes/cluster/gce
   local gci_path
 
+  logecho "Publish release artifacts to gs://$bucket..."
+
+  logecho "- Checking whether $gcs_destination already exists..."
+  if $GSUTIL ls $gcs_destination >/dev/null 2>&1 ; then
+    logecho "- Destination exists. Assuming artifacts at destination are correct. If not, run:"
+    logecho -n "  gsutil -m rm -r $gcs_destination\n"
```

Comment:
Otherwise this annoyingly gets wrapped. I'm not sure why this script bothers to wrap its output, but by default the output was bad without the `-n`.